### PR TITLE
Image preview handler

### PR DESCRIPTION
**Pull request by diego-gw**
Added the option to for the user to provide a handling function 
`imagesPreviewHandler(parsedUrl: string) => string`
when previewing images in the editor allowing dynamic previewing of images directly in the frontend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ easyMDE.value('New input for **EasyMDE**');
 - **indentWithTabs**: If set to `false`, indent using spaces instead of tabs. Defaults to `true`.
 - **initialValue**: If set, will customize the initial value of the editor.
 - **previewImagesInEditor**: - EasyMDE will show preview of images, `false` by default, preview for images will appear only for images on separate lines.
+- **imagesPreviewHandler**: - A custom function for handling the preview of images. Takes the parsed string from the image markdown `![](string)` as argument and returns a string that serves as the `src` attribute of the `<img>` tag in the preview (for ex. as base64). Enables dynamic previewing of images in the frontend without having to upload them to a server.
```
Comment (Ionaru) [inline README.md:147]: I think the `` argument type is a bit confusing. Can you clarify that?

**Review by Ionaru — COMMENTED** [review]
Can you provide an example of how this would be used? I'm confused about what the exact input/output of this function should be.

File: README.md (hunk 1 above)
Comment (diego-gw) [reply README.md]: Tried to clarify it in new commit. Open to comments and suggestions.

**Comment by diego-gw** [comment]
> Can you provide an example of how this would be used? I'm confused about what the exact input/output of this function should be.

When copy-pasting images to the editor, I can use the clipboard event in the frontend to extract the image file and name. Then I can refer to the image in the markdown and use the handling function to preview it immediately. Images can be copy-pasted to the editor in the frontend only and uploaded all at once on save.

Sorry for the late reply.

**Review by Ionaru — APPROVED** [review]

**Pull request merged**